pipelines.go

```diff
@@ -406,3 +406,35 @@ func (s *PipelinesService) DeletePipeline(pid interface{}, pipeline int, options
 
 	return s.client.Do(req, nil)
 }
+
+// UpdateMetadataOptions represents the available UpdateMetadata() options.
+//
+// https://docs.gitlab.com/ee/api/pipelines.html#update-pipeline-metadata
+type UpdateMetadataOptions struct {
+	Name string `url:"ref" json:"name"`
```

Comment:
This needs to be:
```suggestion
	Name *string `url:"name,omitempty" json:"name,omitempty"`
```

Reply:
It is required arguments. Why it will be pointer?

Reply:
If you look around in the code base you'll notice this is the pattern used in all option structs. If we would determine this on a per-field bases it would be a bit nasty to use and (even more so) maintain.

Reply:
Ok. I will prepare patch